storybook/stories/avatarGroup.stories.js

```diff
@@ -0,0 +1,212 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react-native';
+
+import RowView from './RowView';
+import AvatarGroup from '../../src/components/AvatarGroup';
+import { Text } from 'react-native';
+import User from '../../src/components/Icons/user';
+
+const PHOTO = 'https://source.unsplash.com/mEZ3PoFGs_k/50x50';
+
+const avatars = Array.from({ length: 5 }).fill({
+    src: PHOTO,
+});
+
+const avatarsCounter = Array.from({ length: 1000 }).fill({
+    src: PHOTO,
+});
+
+storiesOf('AvatarGroup', module)
+    .add('Basic', () => {
+        return (
+            <React.Fragment>
+                <RowView spaceTop={100} flexDirection={'column'}>
+                    <AvatarGroup
+                        size="large"
+                        avatars={avatars}
+                        maxAvatars={3}
+                        showCounter={false}
+                    />
+                    <Text>Large</Text>
+                </RowView>
+                <RowView spaceTop={30} flexDirection={'column'}>
+                    <AvatarGroup
+                        size="medium"
+                        avatars={avatars}
+                        maxAvatars={3}
+                        showCounter={false}
+                    />
+                    <Text>Medium</Text>
+                </RowView>
+                <RowView spaceTop={30} flexDirection={'column'}>
```

Comment:
here we should make another component instead of add a flexDirection prop because the name of this component is RowView and if it is row then it can't be column